src/flux/store.ts

```diff
@@ -430,6 +465,83 @@
     this.emit("update-flowchart");
   };
 
+  startFlow = async (yamlSTR: string) => {
+    const flowResult = await jinad.startFlow(yamlSTR);
+    if (flowResult.status === "error")
+      return this.showError(`Error starting flow`);
+
+    const { flow_id } = flowResult;
+    _store.activeFlow = flow_id;
```

Comment:
Flux with `store.ts` was the original implementation. Since we moved to the new Redux structure this is no longer relevant. Let's discuss removing flux entirely from the codebase today.